ansible_ai_connect/ai/api/exceptions.py

```diff
@@ -183,3 +183,58 @@ class FeedbackInternalServerException(BaseWisdomAPIException):
     status_code = 500
     default_code = "error__feedback_internal_server"
     default_detail = "Failed to send feedback"
+
+
+class ChatbotNotEnabledException(BaseWisdomAPIException):
+    status_code = 503
+    default_code = "error__chatbot_not_enabled"
+    default_detail = "Chatbot is not enabled"
+
+
+class ChatbotInvalidRequestException(BaseWisdomAPIException):
```

Comment:
Picky; you could extend `WisdomBadRequest`.

https://github.com/ansible/ansible-ai-connect-service/blob/10a81f71d219f30fa749797c5f817ad933accc52/ansible_ai_connect/ai/api/exceptions.py#L69

Reply:
I see...  I will implement the exception as suggested.